src/docs/product/sentry-basics/guides/grouping-and-fingerprints/index.mdx

```diff
@@ -36,7 +36,9 @@ Fortunately you can change the default grouping behavior to make certain issue t
 
 # Solutions
 
-There are three different approaches for updating how events group into issues. The first approach is for merging together historical issues already created. We'll call this **Merging Similar Issues**. The second is for setting rules so next incoming issues will get grouped together. We'll call this **Fingerprint Rules**. The third is **SDK Side Fingerprinting**. The difference between SDK side and Server side is the data elements on the exception and stack traces which you can use for matching issues.
+The easiest way to improve grouping is to [upgrade the project's grouping algorithm](/product/data-management-settings/event-grouping#grouping-algorithms).
```

Comment:
Can you please fix the headings on this page. Solutions should be an H2 rather than an H1 (this is why it's not rendering in the RH sidebar) because the title of the page is the H1. The rest of the headings all need to be moved down a level after that.

Reply:
I would call this out as a beta feature here and not as the "obvious, recommended thing to do to improve grouping". we'd have to backpedal on that level of confidence in further copy anyway

Reply:
@untitaker But upgrading is not always a beta feature, right? E.g. projects that are still on legacy grouping.

I will just revert this change for now.

Reply:
Ah yes you're right. still, would revert for now